config/config_arcaflow.yaml

```diff
@@ -12,7 +12,9 @@ kraken:
     litmus_uninstall_before_run: True                      # If you want to uninstall litmus before a new run starts
     chaos_scenarios:                                       # List of policies/chaos scenarios to load
         -   arcaflow_scenarios:
-            - scenarios/arcaflow/sysbench-cpu-hog/input.yaml
+            - scenarios/arcaflow/io-hog/input.yaml
+            - scenarios/arcaflow/cpu-hog/input.yaml
+            - scenarios/arcaflow/memory-hog/input.yaml
```

Comment:
Missing kill-pod input file.

Reply:
do we want to remove the litmus code as well? (strong +1)